symengine/expression.h

```diff
@@ -305,7 +312,22 @@ struct print_coefficient_impl<U, typename std::
         return os << SymEngine::detail::poly_print(cf);
     }
 };
-}
+} // namespace piranha
 #endif // HAVE_SYMENGINE_PIRANHA
 
+// Utility functions for xeus-cling
+namespace SymEngine
+{
+
+#include <nlohmann/json.hpp>
```

Comment:
Do we need to hook this up into CMake to check that JSON is installed?

Reply:
Fixed with a `#ifdef`